### PR TITLE
MODELIX-456 exportMetaModelFromMps caches too eagerly

### PR DESCRIPTION
**Pull request by mhuster23**
https://issues.modelix.org/issue/MODELIX-456

### CODE REVIEW

File: model-api-gen-gradle/src/main/kotlin/org/modelix/metamodel/gradle/MetaModelGradlePlugin.kt [hunk 1]
```diff
@@ -79,6 +80,7 @@ class MetaModelGradlePlugin: Plugin<Project> {
         }
         val generateMetaModelSources = project.tasks.register("generateMetaModelSources", GenerateMetaModelSources::class.java) {task ->
             task.dependsOn(exportMetaModelFromMps)
+            task.inputs.property("jsonFormatVersion", LanguageData.formatVersion) //necessary for cache invalidation (see https://issues.modelix.org/issue/MODELIX-456)
```
Comment (slisson) [inline model-api-gen-gradle/src/main/kotlin/org/modelix/metamodel/gradle/MetaModelGradlePlugin.kt:83]: There is already a variable `modelixCoreVersion` that you can use instead. See https://github.com/modelix/modelix.core/blob/9ad7a778a4b45ebda67e8259b8ba50df8eae400d/model-api-gen-gradle/src/main/kotlin/org/modelix/metamodel/gradle/MetaModelGradlePlugin.kt#L23

**Review by slisson — CHANGES_REQUESTED** [review]

File: model-api-gen-gradle/src/main/kotlin/org/modelix/metamodel/gradle/MetaModelGradlePlugin.kt (hunk 1 above)
Comment (mhuster23) [reply model-api-gen-gradle/src/main/kotlin/org/modelix/metamodel/gradle/MetaModelGradlePlugin.kt:83]: Yes, but that would invalidate the cache for every new modelix core version, even if the format (or model-api-gen) did not change at all. But using the core version would be better for maintainability.

Comment (slisson) [reply model-api-gen-gradle/src/main/kotlin/org/modelix/metamodel/gradle/MetaModelGradlePlugin.kt:83]: I just realized that this is also the wrong task. `exportMetaModelFromMps` writes the JSON files.

Comment (slisson) [reply model-api-gen-gradle/src/main/kotlin/org/modelix/metamodel/gradle/MetaModelGradlePlugin.kt:83]: All tasks have to rerun if the version changes.

**Review by slisson — APPROVED** [review]

**Pull request merged**